src/components/MovieCard/MovieCard.js

```diff
@@ -1,6 +1,49 @@
 import React from 'react';
+
+import PropsTypes from 'prop-types';
+
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ title, description, imgUrl, imdbUrl }) => (
+  <>
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          src={imgUrl}
+          alt="Film logo"
+        />
+      </figure>
+    </div>
+
+    <div className="card-content">
+      <div className="media">
+        <div className="media-left">
+          <figure className="image is-48x48">
+            <img src="./images/imdb-logo.jpeg" alt="imdb" />
+          </figure>
+        </div>
+
+        <div className="media-content">
+          <p className="title is-8">{title}</p>
+        </div>
+      </div>
+
+      <div className="content">
+        {description}
+        <br />
+        <a href={imdbUrl}>IMDB</a>
+      </div>
+    </div>
+  </>
 );
+
+MovieCard.propTypes = {
+  title: PropsTypes.string.isRequired,
+  description: PropsTypes.string,
+  imgUrl: PropsTypes.string.isRequired,
+  imdbUrl: PropsTypes.string.isRequired,
+};
+
+MovieCard.defaultProps = {
+  description: '',
```

Comment:
```suggestion
  description: 'this movie does not have a description',
```
what user can understand from empty string?